server/package.json

```diff
@@ -13,6 +13,10 @@
     "morgan": "~1.7.0",
     "pug": "~2.0.0-beta10",
     "serve-favicon": "~2.3.2",
+    "graphology": "~0.7.0",
+    "graphology-hits": "~0.1.0",
+    "request": "latest",
+    "fs": "latest",
     "corenlp-client": "latest"
```

Comment:
You should really lock down the versions. Either by not using ~ or ^ or use yarn https://yarnpkg.com/lang/en/